frontend/src/metabase/admin/settings/components/widgets/VersionUpdateNotice/VersionUpdateNotice.tsx

```diff
@@ -26,6 +26,8 @@ export function VersionUpdateNotice() {
   const displayVersion = formatVersion(currentVersion);
 
   if (isHosted) {
+    // FIXME: this is unreachable because we hide the updates section
+    // for cloud customers
     return <CloudCustomers currentVersion={displayVersion} />;
```

Comment:
will sort this out in a separate PR after I get confirmation on the desired behavior